__tests__/app.test.js

```diff
@@ -128,13 +128,36 @@ describe("/api", () => {
       });
       it("404: Not Found when attempting to GET article", () => {
         return supertest(app)
-          .get("/api/article/10000")
+          .get("/api/articles/10000")
           .expect(404)
           .then((res) => {
             const { msg } = res.body;
             expect(msg).toBe("Not Found");
           });
       });
+      it('200: returns a 200 status code, returns article with comment_count included', () => {
+        return supertest(app).get('/api/articles/6?comment_count').expect(200).then(res => {
```

Comment:
the brief for task 12 is that the existing GET /api/articles endpoint has a comment_count property added in the response this is NOT a query so the endpoint in line 139 should still look like '/api/articles'